README.md

```diff
@@ -1,55 +1,185 @@
+* they rarely have any internally changeable `state` properties;
+* and, they are best written as stateless functional components.
+
+Okay so there's our pattern description. Now let's jump into some code and see
+how presentation components actually look in practice.
+
+## Surprise, You've already written presentational components!
+
+Yep, this is true. Think about it. If a presentational component is simply
+a component that doesn't know anything about how to get the data it displays,
+and is mainly responsible for presentation, then you've been doing this from
+the beginning. A simple `HelloWorld` component, for example, is almost
+certainly presentational. Let's see if that's right &mdash; we'll even give our
+component the ability to take a prop:
+
+```javascript
+const HelloWorld = React.createClass({
+  render: function() {
+    return <div class="hello-world">Hello {this.props.message || 'World' }</div>;
+  }
+})
+```
+
+Okay, so does this fit our pattern? Absolutely, it does. Here is a component
+that does nothing but render a piece of our UI; that has no notion of how
+to fetch or reload the `message` data that it takes in as a `prop`; that has
+no changeable state; and that only contains a render method. So, I think we can 
+safely say it's pretty well presentational.
+
+## Okay great, but when would we ever need such a simple component?
 
-We want to start by revisiting something that students have already done
-(without knowing they were doing it): we want to write components that only
-have a `render()` function and just accept props from their parent components.
+Good question! Our `HelloWorld` example is obviously not a very real-world example,
+but consider this: let's say we are working on a massive web application, and we'd
+ like to standardize as well as place some limits on the characteristics of the 
+text inputs used throughout the application's forms.
 
-`<input>`s are actually really great for this:
+In this case, we could certainly establish a style guide in our development team
+that dictates that all uses of `<input>` use a specific set of CSS classes, defined
+in our stylesheets, but this leaves our app open to a lot of human error. People
+would have to consistently follow the convention over time. And while we could
+certainly add props to our inputs by doing something like this --
+`<input className='field' {...props}>` -- we've left the types of props that can
+be provided to our inputs wide open.
+
+With React, we can do much better! Consider this `TextField` component:
 
 ```javascript
 const defaultLimit = 100
 const noop = () => {}
 
 export default class TextField extends React.Component {
   render() {
-    // assume that we have some of these classes defined
-    return <input className="field field-light"
+    // Assume that we have the css classes defined elsewhere.
+    return <input
+             className="field field-light"
              onChange={this.props.onChange || noop}
```

Comment:
I think I'll just cut it. I think it distracts from what we are teaching at the moment.